.scrutinizer.yml

```diff
@@ -0,0 +1,4 @@
+build:
+    dependencies:
```

Comment:
Actually, I'll remove this too in the next commit as it is no longer needed (added it to try and make scrutinizer use the latest setuptools for `ContextualVersionConflict`).